.github/actions/release-image-terraform/action.yml

```diff
@@ -84,27 +76,25 @@ runs:
       id: terraform-apply
       shell: bash
       env:
-        TF_VAR_target_tag: ${{ inputs.apkoTargetTag }}
-        TF_VAR_tag_suffix: ${{ inputs.apkoTargetTagSuffix }}
         APKO_IMAGE: ghcr.io/wolfi-dev/apko:latest@sha256:36e1fdffb0d54aa92bd614eb68f0fe1e4a8e42f939e00e1ea58e2cc2ac1da99d
       run: |
         set -x -o pipefail
         env | grep '^TF_VAR_'
 
-        MODULE=$(basename ${{ inputs.terraformDirectory }})
+        MODULE=${{ inputs.imageName }}
         echo "module=${MODULE}" >> $GITHUB_OUTPUT
 
         # When we have a "mega module", use it to invoke the module
         # for the appropriate image.
         if [ -f "main.tf" ]; then
-          export TF_VAR_target_repository=$(dirname ${{ inputs.apkoBaseTag }})
+          export TF_VAR_target_repository=cgr.dev/chainguard
```

Comment:
This may break downstream rebuilds, we should think about that.

Reply:
Yep, this should be an input to the action.